src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.cs

```diff
@@ -8,6 +8,9 @@ namespace System.IO.Compression
 {
     public static class ZipFile
     {
+        // Per the .ZIP File Format Specification 4.4.17.1 all slashes should be forward slashes
+        private const char c_pathSeparator = '/';
```

Comment:
@ericstj, nit, our design guidelines we've tried to follow in the rest of the repo would have this be PathSeparator rather than c_pathSeparator.